graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/model/GraphQLResponse.kt

```diff
@@ -29,7 +31,14 @@ data class GraphQLResponse(
 )
 
 fun ExecutionResult.toGraphQLResponse(): GraphQLResponse {
-    val filteredErrors = if (errors?.isNotEmpty() == true) errors else null
+    val filteredErrors = if (errors?.isNotEmpty() == true) {
+        errors.map {
+            when (it) {
+                is Exception -> SimpleKotlinGraphQLError(exception = it, locations = it.locations, path = it.path, errorType = it.errorType)
```

Comment:
logic here should be 

1) if it is an instance of `SimpleKotlinGraphQLError` then just return it
2) if it is an Exception then wrap it in `SimpleKotlinGraphQLError
3) otherwise just return the GraphQLError

You need to add the first case as you could implement `SimpleKotlinGraphQLError` that is also an Exception and we don't want to wrap it twice

Reply:
Wow, yeah. Done :)